cmd/gaia/app/genesis.go

```diff
@@ -50,25 +49,15 @@ func (ga *GenesisAccount) ToAccount() (acc *auth.BaseAccount) {
 	}
 }
 
-var (
-	flagName       = "name"
-	flagClientHome = "home-client"
-	flagOWK        = "owk"
-
-	// bonded tokens given to genesis validators/accounts
-	freeFermionVal  = int64(100)
-	freeFermionsAcc = int64(50)
```

Comment:
So I understand that you wanted to have access to these in the server application, but this variable is really application specific, server shouldn't enforce the free fermions provided to the validators and accounts, so this should still be defined at the application level and then passed into the develop server functions

Reply:
Additionally the server should not specify which denom of coin it's giving away - the coin `denom` should also be passed to the server from the application level

Reply:
It actually doesn't even look like these are used in the server at all... just defined!

Reply:
My bad, I moved them back to genesis.go